SPECS-SIGNED/kernel-signed-aarch64/kernel-signed-aarch64.spec

```diff
@@ -82,6 +82,9 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %config %{_localstatedir}/lib/initramfs/kernel/%{uname_r}
 
 %changelog
+* Tue Feb 16 2021 Nicolas Ontiveros <niontive@microsoft.com> - 5.4.91-5
+- Update to kernel release 5.4.91-5.
```

Comment:
Shouldn't this comment, and the one below it, be more explicit about the reason for this change?